packages/tx/src/legacyTransaction.ts

```diff
@@ -356,22 +356,13 @@ export default class Transaction extends BaseTransaction<Transaction> {
   /**
    * Validates tx's `v` value
    */
-  private _validateTxV(v?: bigint, common?: Common): Common {
-    // Check for valid v values in the scope of a signed legacy tx
-    if (v !== undefined) {
-      // v is 1. not matching the EIP-155 chainId included case and...
-      // v is 2. not matching the classic v=27 or v=28 case
-      if (v.ltn(37) && !v.eqn(27) && !v.eqn(28)) {
-        throw new Error(
-          `Legacy txs need either v = 27/28 or v >= 37 (EIP-155 replay protection), got v = ${v}`
-        )
-      }
-    }
-
-    let chainIdBN
```

Comment:
This has been fixed in the "wrong direction", so the removed code is actually the correct one, see #1905 for context and discussion.

So fix should be just to replace `v.ltn(37) && !v.eqn(27) && !v.eqn(28)` with the normal BigInt-operator comparisons and literally nothing else.

(let me know if I overlook something)

Reply:
Update: ah, just seeing that Andrew addressed this in a follow-up commit, so not an issue.